web/js/modules/image-download/util.js

```diff
@@ -143,8 +153,42 @@ export function imageUtilGetCoordsFromPixelValues (pixels, map) {
  */
 export function bboxWMS13(lonlats, crs) {
   if (crs === 'EPSG:4326') {
-    return `${lonlats[0][1]},${lonlats[0][0]},${lonlats[1][1]},${lonlats[1][0]}`;
+    return `${lonlats[0][1]},${lonlats[0][0]},${lonlats[1][1]},${
+      lonlats[1][0]
+    }`;
   } else {
-    return `${lonlats[0][0]},${lonlats[0][1]},${lonlats[1][0]},${lonlats[1][1]}`;
+    return `${lonlats[0][0]},${lonlats[0][1]},${lonlats[1][0]},${
```

Comment:
See comment above

Reply:
That is weird. I'm thinking `prettier.io` must have done that one